src/prefect/orion/schemas/actions.py

```diff
@@ -85,15 +85,15 @@ class DeploymentCreate(ActionBaseModel):
 
     manifest_path: Optional[str] = FieldFrom(schemas.core.Deployment)
     work_queue_name: Optional[str] = FieldFrom(schemas.core.Deployment)
-    worker_pool_name: Optional[str] = Field(
+    work_pool_name: Optional[str] = Field(
```

Comment:
Might want to include a validator in case an old client is sending `worker_pool_name`.